ibmcloud/README.md

```diff
@@ -112,57 +158,79 @@ You need to build a pod VM image for peer pod VMs. A pod VM image contains the f
 * skopeo
 * umoci
 
-The build scripts are located in [ibmcloud/image](./image). The prerequisite software to build a pod VM image is already installed in the worker node by [the Ansible playbook](terraform/cluster/ansible/playbook.yml) for convenience. Note that building a pod VM image on a worker node is not recommended for production, and we need to build a pod VM image somewhere secure to protect workloads running in a peer pod VM.
+The build scripts are located in [ibmcloud/image](./image). The prerequisite software to build a pod VM image is already installed in the worker node by [the Ansible playbook](terraform/cluster/ansible/playbook.yml) for convenience.
 
-The following command builds a custom VM image. A new QCOW2 file with prefix `podvm-` will be created in the current directory.
 
+Note that building a pod VM image on a worker node is not recommended for production, and we need to build a pod VM image somewhere secure to protect workloads running in a peer pod VM.
+
+- SSH to worker node
+```
+ssh root@floating-ip-of-worker-node
+```
+- Go to image folder
 ```
-cd ibmcloud/image
-make build
+cd /root/cloud-api-adaptor/ibmcloud/image
 ```
+- Build a custom VM image. A new QCOW2 file with prefix `podvm-` will be created in the current directory.
+```
+CLOUD_PROVIDER=ibmcloud make build
+```
+
+**Note:** if your worker node is **s390x** based, the suffix of this new created QCOW2 file will be `-s390x` otherwise it will be `-amd64`.
+
 
 You need to configure Cloud Object Storage (COS) to upload your custom VM image.
 
 https://cloud.ibm.com/objectstorage/
 
+
 First, create a COS service instance if you have not create one. Then, create a COS bucket with the COS instance. The COS service instance and bucket names are necessary to upload a custom VM image.
 
-The following environment variables are necessary to be set before executing the image upload script. You can change IBMCLOUD_COS_REGION if you prefer another region. In this case, you also want to change IBMCLOUD_COS_SERVICE_ENDPOINT to one of endpoints listed at [https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints).
+
+The following environment variables are necessary to be set before executing the image upload script. You can change `IBMCLOUD_COS_REGION` if you prefer another region. In this case, you also want to change `IBMCLOUD_COS_SERVICE_ENDPOINT` to one of endpoints listed at [https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints).
 
 ```
 export IBMCLOUD_API_KEY=<your API key>
 export IBMCLOUD_COS_SERVICE_INSTANCE=<COS service instance name>
 export IBMCLOUD_COS_BUCKET=<COS bucket name>
+export IBMCLOUD_API_ENDPOINT=https://cloud.ibm.com
+export IBMCLOUD_VPC_REGION=us-south
```

Comment:
I just wanted to check if `IBMCLOUD_VPC_REGION` should be `jp-tok` in this documentation to match the rest of the defaults?

Reply:
Thanks, a typo, fixed.